package/gluon-core/luasrc/lib/gluon/upgrade/180-outdoor-preset

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/lua
+
+-- This script needs to be sorted before 200-wireless as it affects
+-- wireless channel selection and wireless mesh configuration.
+
+local uci = require('simple-uci').cursor()
+
+if uci:get('gluon', 'wireless', 'outdoor') ~= nil then
+	os.exit(0)
+end
+
+local platform_info = require 'platform_info'
+
+uci:section('gluon', 'wireless', 'wireless', {
+    outdoor = platform_info.is_outdoor_device()
+})
```

Comment:
Fixed in 492c22e092cd39e5fadf7efd49412d11412f54c0

Reply:
1. Rather than doing nothing when sysconfig.gluon_version is set, outdoor should be set to false in this case
2. I think it would make sense to have a site.conf option that can force the default to on or off

Reply:
1. Wow, I didn't properly read what you wrote earlier, sorry. Fixed in f6b9febb436ee39e4db64c4ddac09155f6226ad2.
2. TODO.

Reply:
2. https://github.com/freifunk-gluon/gluon/pull/1329/commits/662b84ec82db95a0afe7806801fbaa8ff94d6a68